<launch>
        <node pkg="dr_controller" type="dr_real_driver.py" name="dr_driver" output="screen">
            <param name="dr_ip" value="192.168.1.101"/>
            <param name="dr_password" value="uGRqirr3"/>
            <param name="upper_deadzone" value="0.3"/>
            <param name="lower_deadzone" value="-0.3"/>
        </node>

        <node type="ackermann_translator.py" name="ackermann_translator" pkg="dr_controller" output="screen">
            <param name="wheelbase" value="0.16"/>
        </node>

        <node name="drive_pid" pkg="pid" type="controller" ns="drive_pid">
          <param name="Kp" value="2" />
          <param name="Ki" value="0.3" />
          <param name="Kd" value="0.0" />
          <param name="rate" value="50" />
          <param name="upper_limit" value="1.0" />
          <param name="lower_limit" value="-1.0" />
          <param name="windup_limit" value="0.8" />
        </node>

</launch>